GettingStarted/ibmdbpy_GettingStarted_1-basics.ipynb

```diff
@@ -0,0 +1,2489 @@
+       "    }\n",
+       "\n",
+       "    .dataframe thead th {\n",
+       "        text-align: right;\n",
+       "    }\n",
+       "</style>\n",
+       "<table border=\"1\" class=\"dataframe\">\n",
+       "  <thead>\n",
+       "    <tr style=\"text-align: right;\">\n",
+       "      <th></th>\n",
+       "      <th>sepal_length</th>\n",
+       "      <th>sepal_width</th>\n",
+       "      <th>petal_length</th>\n",
+       "      <th>petal_width</th>\n",
+       "      <th>species</th>\n",
+       "    </tr>\n",
+       "  </thead>\n",
+       "  <tbody>\n",
+       "    <tr>\n",
+       "      <th>145</th>\n",
+       "      <td>7.7</td>\n",
+       "      <td>3.0</td>\n",
+       "      <td>6.1</td>\n",
+       "      <td>2.3</td>\n",
+       "      <td>virginica</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>146</th>\n",
+       "      <td>7.7</td>\n",
+       "      <td>2.6</td>\n",
+       "      <td>6.9</td>\n",
+       "      <td>2.3</td>\n",
+       "      <td>virginica</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>147</th>\n",
+       "      <td>7.7</td>\n",
+       "      <td>3.8</td>\n",
+       "      <td>6.7</td>\n",
+       "      <td>2.2</td>\n",
+       "      <td>virginica</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>148</th>\n",
+       "      <td>7.7</td>\n",
+       "      <td>2.8</td>\n",
+       "      <td>6.7</td>\n",
+       "      <td>2.0</td>\n",
+       "      <td>virginica</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>149</th>\n",
+       "      <td>7.9</td>\n",
+       "      <td>3.8</td>\n",
+       "      <td>6.4</td>\n",
+       "      <td>2.0</td>\n",
+       "      <td>virginica</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "     sepal_length  sepal_width  petal_length  petal_width    species\n",
+       "145           7.7          3.0           6.1          2.3  virginica\n",
+       "146           7.7          2.6           6.9          2.3  virginica\n",
+       "147           7.7          3.8           6.7          2.2  virginica\n",
+       "148           7.7          2.8           6.7          2.0  virginica\n",
+       "149           7.9          3.8           6.4          2.0  virginica"
+      ]
+     },
+     "execution_count": 9,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# Take a look at the last records\n",
+    "idadf.tail()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Note: Running the two above cells on a distributed system would imply that you may not always get the same results, unless using an `ORDER BY` clause. An implicit sorting is therefore operated to better mimic the behaviour of Pandas. This sorting can be disabled by setting the option `sort`to False in `head()` and `tail()`."
```

Comment:
Going into this kind of details may confuse the reader don't you think? It's like saying "by the way sorry nothing is guaranteed".
But if you find the original sentence shocking I will rephrase it.

Reply:
There's no distinction between distributed and non-distributed systems, so it's not necessary to go into that much detail. Something like "You can see in the two previous cells that the `head` and `tail` functions sort the data by the first numerical column. This can be disabled by setting the option `sort`to False when calling them. Other functions do not automatically sort the data." would be good enough, IMHO.

Reply:
ok I agree 👍